src/makeRoutes.js

```diff
@@ -1,12 +1,18 @@
 import userRouter from './routes/user';
 import userAuthRouter from './routes/user/auth';
+import NotificationTypeController from './controllers/NotificationTypeController';
```

Comment:
Should be importing the router for NotificationTypes just like we are for Users instead of the Controller like you're doing.

Reply:
done